examples/ccvm_boxqp_plot.py

```diff
@@ -29,8 +29,20 @@
 
     # Supply solver parameters for different problem sizes
     solver.parameter_key = {
-        10: {"pump": 1.0, "dt": 0.001, "iterations": 10000, "noise_ratio": 15},
-        20: {"pump": 2.0, "dt": 0.005, "iterations": 15000, "noise_ratio": 10},
+        10: {
+            "pump": 1.0,
```

Comment:
This will temporarily get rid of the error:
            "pump": 8.0,
```

Reply:
Awesome, thank you!!
You're right, I do have other issues created to address the rest of the problem separately, but this is a great start to get this script running :)